### PR TITLE
Update data for recent Duktape versions

### PR DESCRIPTION
**Pull request by omasanori**

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: environments.json [hunk 1]
```diff
@@ -4140,6 +4140,30 @@
     "short": "DUK 2.3",
     "platformtype": "engine",
     "release": "2017-09-23",
+    "obsolete": true
```
Comment (afmenez) [inline environments.json]: This is more than 4 years old, shouldn't it be "very" instead of true?

Comment (omasanori) [reply environments.json]: Thank you for reviewing! OK, I was not sure if it is very obsolete or just obsolete.

**Comment by omasanori** [comment]
Changed all v1 versions to `"obsolete": "very"` as `v1-maintenance`  branch is inactive now. Also, all v2 versions released in 2018 or earlier are marked obsolete.

**Comment by afmenez** [comment]
Great, but you need to rebuild for this to take effect.

**Comment by omasanori** [comment]
Good catch! Rebuilt the pages.

**Comment by omasanori** [comment]
Rebased.

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**